test/evergreen.yml

```diff
@@ -5875,6 +5885,10 @@ buildvariants:
     - name: examples-c-production-disable-static-test
     - name: format-stress-pull-request-test
     - name: make-check-test
+      distros: ubuntu2004-large
+    - name: csuite-long-running
+      distros: ubuntu2004-large
+      exec_timeout_secs: 21600 # 6 hrs
```

Comment:
The project-level setting is 6 hrs (see the "Project Settings" section), making this line of setting unnecessary.

Reply:
Ah great, I'll remove this